source/_components/light.mqtt.markdown

```diff
@@ -167,7 +171,7 @@ rgb_state_topic:
   required: false
   type: string
 rgb_value_template:
-  description: "Defines a [template](/docs/configuration/templating/#processing-incoming-data) to extract the RGB value."
+  description: "Defines a [template](/docs/configuration/templating/) to extract the RGB value."
```

Comment:
This should still link to /docs/configuration/templating/#processing-incoming-data, change the link from `rgb_command_template instead`.